core-ui/src/shared/components/ReadonlyEditorPanel.js

```diff
@@ -10,6 +10,9 @@ export function ReadonlyEditorPanel({ title, value, editorProps }) {
     minimap: {
       enabled: false,
     },
+    scrollbar: {
```

Comment:
We have more Monaco editors in the app (like lambda details view), we should add it everywhere.